src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php

```diff
@@ -128,6 +128,7 @@ public function configureViews(ViewCollection $viewCollection): void
                     ->setListKey('contacts')
                     ->setTitle('sulu_contact.people')
                     ->addListAdapters(['table'])
+                    ->setItemDisabledCondition('lastName == "Test"')
```

Comment:
I think that should be gone before the merge, otherwise it will result in very confusing behavior :see_no_evil:

Reply:
😂